ci-operator/step-registry/openshift/e2e/openstack/proxy/openshift-e2e-openstack-proxy-workflow.yaml

```diff
@@ -33,7 +33,13 @@ workflow:
         DNS should provide DNS for the cluster \|
         DNS should provide /etc/hosts entries for the cluster \|
         DNS should provide DNS for pods for Subdomain \|
-        DNS should provide DNS for pods for Hostname
+        DNS should provide DNS for pods for Hostname \|
+        DNS should resolve DNS of partial qualified names for the cluster \|
```

Comment:
Do we know why these tests started failing? Have we opened a BZ?

I'm generally not in favor of skipping tests without a very good justification, and plans to either fix the test to work in $environment or have the test suite skip the test if it does not apply to the environment.

In other words we shouldn't abuse `TEST_SKIPS` and leave that for temporary skipping tests.

Reply:
If the tests must be skipped at a higher level, they can be added to the `[Skipped:Proxy]` rule:

https://github.com/openshift/origin/blob/2a2e6c9/test/extended/util/annotate/rules.go#L98-L101